backstop/tests/side-navigation.html

```diff
@@ -95,131 +103,261 @@ <h1>Side navigation</h1>
     <hr />
 
     <section id="demo-side-navigation">
-      <div
```

Comment:
Why you moved expander to the bottom?

Reply:
I placed the expander on the bottom because the header content of the submenu (back button, title, action buttons) are vertically aligned with the first button:
![Screen Shot 2021-09-21 at 9 21 49 AM](https://user-images.githubusercontent.com/849817/134178280-14fa24c8-59a0-400b-bef8-f44911c4b488.png)

When the expander is at the top, the alignment doesn't look as nice:
![Screen Shot 2021-09-21 at 9 23 22 AM](https://user-images.githubusercontent.com/849817/134178551-957084aa-92e9-4271-8c17-3b6db467e939.png)